operations/sort_results_edge_attribute.yml

```diff
@@ -0,0 +1,42 @@
+id: sort_results_edge_attribute
+subclass_of: sort_results
+name: sort results (edge attribute)
+description: This operation sorts the results by the given edge attribute. If in ascending order the maximum value of the results edges with the given attribute will be taken while the minimum will be taken for descending order. If the result has no edges with te given attribute it will be listed last. If `max_results` is given the it truncates the results to at most the given value.
+examples:
+- input: sort/messages/01_results_full.json
+  output: sort/messages/02_sorted_results_edge_attribute.json
+  parameters:
+    edge_attribute: normalized_google_distance
+    ascending_or_descending: ascending
+    max_results: 2
+input_requirements: []
+output_guarantees:
+- max `max_results` results
+allowed_changes:
+- change results
+- remove results
+parameters:
+  type: object
+  properties:
+    edge_attribute:
+      description: The name of the edge attribute to order by.
+      type: string
+      example: normalized_google_distance
+    ascending_or_descending:
+      description: Indicates whether results should be sorted in ascending or descending order.
+      type: string
+      enum:
+        - ascending
+        - descending
+    qedge_keys:
+      description: This indicates if you only want to consider edges with specific edge_keys. If not provided or empty, all edges will be looked at.
+      type: array
+      example: [e01]
+    max_results:
```

Comment:
Is using `max_results` equivalent to using the operations`filter_results_top_n`? Do we need to do it here?

Reply:
It is equivalent. I suppose we so not need to do it here as it would be the same as sorting however you want then using `filter_results_top_n`.

Reply:
This has now been removed.